app/scripts/styles/_uswds-theme.scss

```diff
@@ -8,12 +8,74 @@
             "display-name": var(--base-font-family),
             "cap-height": 364px,
         ),
+        "system": (
+            "cap-height": 364px
+        ),
+        "merriweather": (
+            display-name: "Merriweather Web",
+            cap-height: 364px,
```

Comment:
I couldn't find a way to only set the cap-height without adding the rest of the values for the typeface, copied from the uswds config defaults.